src/utility/SuggestionsManager.ts

```diff
@@ -16,86 +18,162 @@ export abstract class SuggestionsManager {
      */
     /*private static _suggestions: Collection<number, string> = new Collection<number, string>();*/
 
-    private static _suggestions: Array<string> = new Array<string>();
+    // private static _suggestions: Array<string> = new Array<string>();
+    private static _suggestions: Collection<string, Array<string>> = new Collection<string, Array<string>>();
+    private static _suggestions2: Object = { string: Array<string>()};
```

Comment:
It doesn't look like this is being used?

Reply:
You right. It was being used during testing. Cleaning up.